lib/features/email/presentation/model/message_content.dart

```diff
@@ -9,6 +11,23 @@ class MessageContent with EquatableMixin {
 
   MessageContent(this.textFormat, this.content);
 
+  bool hasImageInlineWithCid() => content.contains('cid:');
+
+  String getContentHasInlineAttachment(Uri baseDownloadUrl, AccountId accountId, List<AttachmentFile> attachmentFiles) {
+    var contentValid = content;
+    attachmentFiles.forEach((attachment) {
+      if(attachment.cid != null) {
+        final urlDownloadImage = baseDownloadUrl.origin
+            .replaceAll('{accountId}', '${accountId.id.value}')
+            .replaceAll('{blobId}', '${attachment.blobId?.value}')
+            .replaceAll('{type}', '${attachment.type?.mimeType}')
+            .replaceAll('{name}', '${attachment.name}');
+        contentValid = content.replaceAll('cid:${attachment.cid}', '$urlDownloadImage');
```

Comment:
please using `UriTemplate`: https://pub.dev/documentation/uri/latest/uri/UriTemplate-class.html. And please create method to create download URL(Maybe in Session?).
```dart 
Url getDownloadUrl(String accountId, String blobId, String name, String type)
```